docs/advanced.rst

```diff
@@ -1019,104 +1143,651 @@ to learn more about the use of this feature.
++--------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp             |
+|    :language: c++                                      |
+|    :start-after: //CONF_QOS_STATIC_DISCOVERY_USERID    |
+|    :end-before: //!                                    |
++--------------------------------------------------------+
+| **XML**                                                |
++--------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml              |
+|    :language: xml                                      |
+|    :start-after: <!-->CONF_QOS_STATIC_DISCOVERY_USERID |
+|    :end-before: <!-->                                  |
++--------------------------------------------------------+
+
+On the local participant, loading STATIC EDP configuration files is done by:
+
++---------------------------------------------------------+
+| **C++**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/CodeTester.cpp              |
+|    :language: c++                                       |
+|    :start-after: //CONF_STATIC_DISCOVERY_XML            |
+|    :end-before: //!                                     |
++---------------------------------------------------------+
+| **XML**                                                 |
++---------------------------------------------------------+
+| .. literalinclude:: ../code/XMLTester.xml               |
+|    :language: xml                                       |
+|    :start-after: <!-->CONF_STATIC_DISCOVERY_XML         |
+|    :end-before: <!-->                                   |
++---------------------------------------------------------+
+
+.. role:: raw-html(raw)
+    :format: html
+
+.. _discovery_server:
+
+Server-Client Discovery
+=======================
+
+This mechanism is based on a client-server discovery paradigm, i.e. the metatraffic (message exchange among participants
+to identify each other) is centralized in one or several server participants (left figure), as opposed to simple
+discovery (right figure), where metatraffic is exchanged using a message broadcast mechanism like an IP multicast
+protocol.
+
+.. image:: ds_uml.png
+    :align: center
+    :width: 50%
+
+.. _DS_key_concepts:
+
+Key concepts
+------------
+
+In this architecture there are several key concepts to understand:
+
+- The Server-client discovery mechanism reuses the RTPS discovery messages structure, as well as the standard RTPS
+  writers and readers.
+
+- Discovery server participants may be *clients* or *servers*. The only difference between them is on how the
+  meta-traffic is handled. The user traffic, that is, the traffic among the publishers and subscribers they create is
+  role-independent.
+
+- All *server* discovery info will be shared with their linked *clients* and likewise the own *clients* discovery info
```

Comment:
```suggestion
- All *server* discovery info will be shared with their linked *clients* and likewise the *clients* discovery info
```

Reply:
mmm, here *own* is used to emphasize that is each *client* info as participant. I think we lose clarity by removing it.

Reply:
I have a different suggestion
```suggestion
- All *server* and *client* discovery info will be shared with linked *clients*.
```